evm_loader/api/src/api_server/handlers/trace.rs

```diff
@@ -1,35 +1,37 @@
-use axum::{http::StatusCode, Json};
 use std::convert::Into;
 
+use actix_web::{http::StatusCode, post, web, Responder};
+
 use crate::{context, types::request_models::TraceRequestModel, NeonApiState};
 
 use super::{parse_emulation_params, process_error, process_result};
 
-#[allow(clippy::unused_async)]
+#[post("/trace")]
```

Comment:
In part of paths you use just name, but here with slash. Please, make them the same.